docs/mkdocs.yml

```diff
@@ -362,6 +362,7 @@ nav:
         - folder move: 'cmd/spo/folder/folder-move.md'
         - folder remove: 'cmd/spo/folder/folder-remove.md'
         - folder rename: 'cmd/spo/folder/folder-rename.md'
+        - folder rename: 'cmd/spo/folder/folder-roleassignment-add.md'
```

Comment:
Please update the description

Reply:
Copy and paste without completetly amending the description:< .  Thanks for the review. Removing the local cli instance and trying any m365 commands was still failing. I deleted all docker instances and started from scratch to get rid of the error message.